code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2936,14 +2942,28 @@
 	if((reac_volume <= 10)) // Makes sure people don't duplicate ants.
 		return
 
-	var/obj/effect/decal/cleanable/ants/pests = exposed_turf.spawn_unique_cleanable(/obj/effect/decal/cleanable/ants)
+	var/obj/effect/decal/cleanable/ants/pests = exposed_turf.spawn_unique_cleanable(ants_decal)
 	if(!pests)
 		return
 
 	var/spilled_ants = (round(reac_volume,1) - 5) // To account for ant decals giving 3-5 ants on initialize.
-	pests.reagents.add_reagent(/datum/reagent/ants, spilled_ants)
+	pests.reagents.add_reagent(src.type, spilled_ants)
```

Comment:
```suggestion
	pests.reagents.add_reagent(type, spilled_ants)
```

Reply:
the argument is `reagent_type` here so you're safe for just `type`, had to double check